docs/source/documentation/platform/index.html.md.erb

```diff
@@ -0,0 +1,12 @@
+---
+owner_slack: "#data-platform-notifications"
+title: Data Platform documentation
+last_reviewed_on: 2023-03-1
+review_in: 6 months
+---
+
+# Data Platform documentation
+
+Here you will find introductory documentation to the Data Platform, as well as technical documentation.
+
+[What is the Data Platform](/documentation/platform/what-is-the-data-platform.html)
```

Comment:
Should this be a relative link to `./platform/what-is-the-data-platform.html` ?

Reply:
Probably, but I don't really know. I'd like to get it published and learn what happens. I actually think the right link might be `./what-is-the-data-platform.html` since they are in the same directory

Reply:
it needs to be the full path all the time, a quirk of tech-docs